docs/source/faqs.md

```diff
@@ -183,4 +183,39 @@ This has been solved in `ray>=2.47.1`, thus we could solve this as following:
+(EngineCore_0 pid=519998) ERROR 08-05 11:23:25 [core.py:632] RuntimeError: capture_begin:build/CMakeFiles/torch_npu.dir/compiler_depend.ts:156 NPU function error: c10_npu::acl::AclmdlRICaptureBegin(capture_stream_, capture_mode), error code is 207005
+(EngineCore_0 pid=519998) ERROR 08-05 11:23:25 [core.py:632] [ERROR] 2025-08-05-11:23:25 (PID:519998, Device:0, RankID:-1) ERR00100 PTA call acl api failed
+(EngineCore_0 pid=519998) ERROR 08-05 11:23:25 [core.py:632] [Error]: Failed to apply for memory.
+(EngineCore_0 pid=519998) ERROR 08-05 11:23:25 [core.py:632]         Check the remaining storage space in the hardware environment.
+(EngineCore_0 pid=519998) ERROR 08-05 11:23:25 [core.py:632] EL0006: [PID: 519998] 2025-08-05-11:23:25.691.093 The resources are insufficient.
+(EngineCore_0 pid=519998) ERROR 08-05 11:23:25 [core.py:632]         Solution: Close applications not in use.
+(EngineCore_0 pid=519998) ERROR 08-05 11:23:25 [core.py:632]         TraceBack (most recent call last):
+(EngineCore_0 pid=519998) ERROR 08-05 11:23:25 [core.py:632]         [SqCqManage]Alloc sq cq fail, stream_id=1984, retCode=0x7020023.[FUNC:AllocStreamSqCq][FILE:stream_sqcq_manage.cc][LINE:78]
+(EngineCore_0 pid=519998) ERROR 08-05 11:23:25 [core.py:632]         [SqCqManage]Alloc sq cq fail, stream_id=1984, retCode=0x7020023.[FUNC:Setup][FILE:stream.cc][LINE:655]
+(EngineCore_0 pid=519998) ERROR 08-05 11:23:25 [core.py:632]         rtStreamBeginCapture execute failed, reason=[driver error:resource alloc fail][FUNC:FuncErrorReason][FILE:error_message_manage.cc][LINE:53]
+(EngineCore_0 pid=519998) ERROR 08-05 11:23:25 [core.py:632]         begin capture stream failed, runtime result = 207005[FUNC:ReportCallError][FILE:log_inner.cpp][LINE:161]
+(EngineCore_0 pid=519998) ERROR 08-05 11:23:25 [core.py:632]         [SqCqManage]Alloc sq cq fail, stream_id=1985, retCode=0x7020023.[FUNC:AllocStreamSqCq][FILE:stream_sqcq_manage.cc][LINE:78]
+(EngineCore_0 pid=519998) ERROR 08-05 11:23:25 [core.py:632]         [SqCqManage]Alloc sq cq fail, stream_id=1985, retCode=0x7020023.[FUNC:Setup][FILE:stream.cc][LINE:655]
+(EngineCore_0 pid=519998) ERROR 08-05 11:23:25 [core.py:632]         stream setup failed, retCode=0x7020023.[FUNC:SyncGetDevMsg][FILE:api_impl.cc][LINE:6109]
+(EngineCore_0 pid=519998) ERROR 08-05 11:23:25 [core.py:632]         Sync get device msg failed, retCode=0x7020023.[FUNC:GetDevErrMsg][FILE:api_impl.cc][LINE:6159]
+(EngineCore_0 pid=519998) ERROR 08-05 11:23:25 [core.py:632]         rtGetDevMsg execute failed, reason=[driver error:resource alloc fail][FUNC:FuncErrorReason][FILE:error_message_manage.cc][LINE:53]
+(EngineCore_0 pid=519998) ERROR 08-05 11:23:25 [core.py:632]
+```
+
+Try to adjust the arg `cuda-capture-sizes` to address this. This is more likely to happen when you're using A3. Please refer to the empirical formula below to estimate a suitable value for this argument:
```

Comment:
There mentioned two configuration `cuda-capture-sizes` and `num_capture_sizes`, the users might confused which one should be used and how to used.